zetta_utils/mazepa_layer_processing/common/subchunkable_apply_flow.py

```diff
@@ -50,144 +52,164 @@ def expand_if_singleton(arg: Union[T, List[T]], length: int) -> List[T]:
 
 
 @builder.register("build_subchunkable_apply_flow")
-def build_subchunkable_apply_flow(  # pylint: disable=keyword-arg-before-vararg, line-too-long, too-many-locals, too-many-branches
-    bbox: BBox3D,
+def build_subchunkable_apply_flow(  # pylint: disable=keyword-arg-before-vararg, line-too-long, too-many-locals, too-many-branches, too-many-statements
+    start_coord: IntVec3D,
+    end_coord: IntVec3D,
+    coord_resolution: Vec3D,
```

Comment:
If subchunkable is created directly by the user, then this is great. 
But if it's created from within another flow ([[eg](https://github.com/ZettaAI/zetta_utils/blob/main/zetta_utils/mazepa_layer_processing/alignment/compute_field_flow.py#L158)]), converting `idx.bbox` to `start_coord`, `end_coord` and `coord_resolution` doesn't sound clean. 

Perhaps `build_subchunkable_apply_flow_shortcut` can be the one registered with the builder and take in coords, build coords into a bbox and pass it into `build_subchunkable_apply_flow`?

If this has non-trivial concerns/consequences, we can defer it to a separate PR after this is merged.

Reply:
I think having "shortcut" sounds even messier - I'll change it so that it can gracefully accept either `bbox` or `start_coord, end_coord, coord_resolution`.